dojo/unittests/tools/test_burp_graphql_parser.py

```diff
@@ -0,0 +1,78 @@
+from os import path
+
+from django.test import TestCase
+from dojo.models import Test
+from dojo.tools.burp_graphql.parser import BurpGraphQLParser
+
+
+class TestBurpGraphQLParser(TestCase):
+
+    def test_burp_one_finding(self):
+        with open(path.join(path.dirname(__file__), "../scans/burp_graphql/one_finding.json")) as test_file:
+            parser = BurpGraphQLParser()
+            findings = parser.get_findings(test_file, Test())
+            for finding in findings:
+                for endpoint in finding.unsaved_endpoints:
+                    endpoint.clean()
+
+            self.assertEqual(1, len(findings))
+            self.assertEqual("Finding", findings[0].title)
+            self.assertEqual('79', findings[0].cwe)
+            self.assertIn("description 1", findings[0].description)
+            self.assertIn("remediation 1", findings[0].mitigation)
+            self.assertIn("issue description 1", findings[0].impact)
+            self.assertIn("issue remediation 1", findings[0].mitigation)
+            self.assertEquals('High', findings[0].severity)
+            self.assertEqual(1, len(findings[0].unsaved_endpoints))
+            self.assertEqual('https://www.test.com/path', str(findings[0].unsaved_endpoints[0]))
```

Comment:
better check the attribute of the endpoint here

Reply:
Updated in latest commit (361b5f2)
Changed the test to check the attributes of the endpoint object